code/datums/status_effects/debuffs/debuffs.dm

```diff
@@ -334,6 +334,47 @@
 /datum/status_effect/pacify/on_remove()
 	REMOVE_TRAIT(owner, TRAIT_PACIFISM, "status_effect")
 
+/datum/status_effect/demonic_hallcuination
+	id = "demonic_hallcuination"
```

Comment:
you've mispelled hallucination all over the place

Reply:
guh